tests/app/config/pimcore/classes/definition_TestDataObject.php

```diff
@@ -0,0 +1,136 @@
+<?php declare(strict_types=1);
+
+return Pimcore\Model\DataObject\ClassDefinition::__set_state([
+    'dao' => null,
+    'id' => '13',
```

Comment:
Probably it doesn't matter, but maybe we should use `test_data_object` as ID:

```suggestion
    'id' => 'test_data_object',
```